a4c-brooklyn-plugin/src/main/resources/brooklyn/brooklyn-matching-config.yaml

```diff
@@ -0,0 +1,16 @@
+
+matching_configurations:
+  brooklyn.nodes.Compute:
+    capabilities:
+      - host:
+          properties:
+            - num_cpus: { less_or_equal: 0 } # we match if the template value is less or equal that the location resource value
```

Comment:
This deserves more clarification. It's unclear how the `less_or_equal` corresponds to the comment.